lib/structuraid_core/designers/footings/steps/centric_isolated/set_reinforcement_layers_coordinates_to_footing.rb

```diff
@@ -0,0 +1,101 @@
+require 'interactor'
+require 'matrix'
+
+module StructuraidCore
+  module Designers
+    module Footings
+      module Steps
+        module CentricIsolated
+          # Runs the structural bending analysis and design for a centric isolated footing, generating the required reinforcement ratio and checking that the provided one is enough
```

Comment:
Is this the right description of this class? It looks like you copied it and pasted it

Reply:
Done